cpp/tests/io/parquet_test.cpp

```diff
@@ -4465,6 +4471,138 @@ TEST_F(ParquetWriterTest, CheckColumnOffsetIndexStruct)
   }
 }
 
+TEST_F(ParquetWriterTest, CheckColumnIndexListWithNulls)
+{
+  auto null_at_even_idx =
```

Comment:
It looks like the places where this is used are only nulling out one or two values. Could we just write `using cudf::test::iterators::nulls_at;` and then `nulls_at({0, 2})` or similar in those locations? If there was a longer list, then I would support using an iterator, but not for 1-2 values where being explicit is still concise. I don't feel strongly here, so we could leave this as-is if you prefer.